### PR TITLE
test metrics: Add script to measure kernel boot time and fix memory usage script

### PR DESCRIPTION
**Pull request by chavafg**
This PR contains 2 commits:
1. tests: add script to measure kernel boot time.
With this script, we will be able to measure the kernel
boot time when a container is launched using cc-oci-runtime.
The script uses dmesg log to get the data

2. tests: fixed condition to check if smem is installed
docker_memory_usage.sh needs smem tool to be installed in the system.
this commit fixes the condition that checks if smem is installed.

### CODE REVIEW

**Comment by chavafg** [comment]
qa-passed

**Comment by dlespiau** [comment]
lgtm

File: tests/metrics/density/docker_memory_usage.sh.in [hunk 1]
```diff
@@ -53,8 +53,8 @@ function get_docker_memory_usage(){
 }
 
 echo "Executing Test: ${TEST_NAME}"
-if [ ! -f $SMEM_BIN ]; then
-	die "${SMEM_BIN} is not installed in your system, skipping test"
+if [ "$SMEM_BIN" == "" ]; then
+	die "smem is not installed in your system, skipping test"
```
Comment (gorozco1) [inline tests/metrics/density/docker_memory_usage.sh.in]: Is then smem part of .travis_setup.sh ?

Comment (gorozco1) [inline tests/metrics/density/docker_memory_usage.sh.in]: oh.. ok then...

**Comment by gorozco1** [comment]
lgtm

**Pull request merged**